app/controllers/api/users_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::UsersController < ApplicationController
+  def posts
+    @posts = User.find(params[:id]).posts
+    render json: { posts: @posts }
+  end
+end
```

Comment:
- [ ] [OPTIONAL] Great job team !, however, there are couple of improvements you might consider.
-  I changed `User.find(params[:id]).posts` to `@user.posts` for better readability.
- I also added an instance variable `@user` to make it accessible in case you need it for further actions or rendering. You can implement this in the User controller as well.
``` ruby
class Api::UsersController < ApplicationController
  def posts
    @user = User.find(params[:id])
    @posts = @user.posts
    render json: { posts: @posts }
  end
end
```

Reply:
Hi @Whoistolu 
Noted and thank you very much for your insightful review and assistance in refactoring our code for readability.